python/lsst/pipe/tasks/colorterms.py

```diff
@@ -166,7 +166,7 @@ def getColorterm(self, filterName, refCatName, doRaise=True):
             ctDictConfig = self.data.get(refCatName)
             if ctDictConfig is None:
                 # try glob expression
-                matchList = [glob for glob in self.data if fnmatch.fnmatch(refCatName, glob)]
+                matchList = [glob for glob in self.data if fnmatch.fnmatch(glob, refCatName)]
```

Comment:
I agree the variable needs to be renamed and I think Simon is right that the argument order is wrong. The problem is that I thought I could use a glob for config.refCatName but I can't now and probably shouldn't be able to; the globs are the catalog names in ColortermLibary (the keys of that dict).

This whole technique for managing color term data is a dreadful hack, but until we have better management of calibration data I doubt it's worth trying to clean it up beyond fixing the bug Simon noted.